suites/Git_Import_Test.py

```diff
@@ -0,0 +1,40 @@
+import sys
+from helpers import base
+import lemoncheesecake.api as lcc
+from lemoncheesecake.matching import *
+from pages import git_import_page
+from pages import search_page
+from helpers import utilities
+from helpers import constants
+from helpers import locators
+sys.path.append("..")
+
+SUITE = {
+    "description": "Git import for authenticated user",
+    "rank": "2"
+}
+git_import_repo_URL = base.config_reader('git_import_test_repo', 'git_import_repo_url')
+git_import_repo_Name = base.config_reader('git_import_test_repo', 'git_import_repo_name')
+
+@lcc.test("Warning message should be displayed when repository URL is empty")
+def git_import_for_empty_git_repo(driver):
+    utilities.click_element_by_link_text(driver, locators.MENU_GIT_IMPORT_LINK_TEXT)
+    git_import_page.import_git_repo(driver, None, None)
+    check_that("Empty git repo url warning message", utilities.get_text_by_css(driver, locators.REPO_URL_EMPTY_WARNING_CSS), contains_string(constants.repo_url_empty_warning_message))
```

Comment:
break it into two lines of text, it is exceeding 120 -140 characters. Same for the ones below also.